src/legacy/core_plugins/console/np_ready/public/application/containers/editor/legacy/use_ui_ace_keyboard_mode.tsx

```diff
@@ -99,7 +99,6 @@ export function useUIAceKeyboardMode(aceTextAreaElement: HTMLTextAreaElement | n
       if (aceTextAreaElement) {
         document.removeEventListener('keydown', documentKeyDownListener);
         aceTextAreaElement.removeEventListener('keydown', aceKeydownListener);
-        document.removeChild(overlayMountNode.current!);
```

Comment:
Why is this being removed?

Reply:
I was running into the first of these two issues: https://developer.mozilla.org/en-US/docs/Web/API/Node/removeChild#Errors_thrown

Which indicated that cleanup for this DOM node is happening with the react component unmounting.

Reply:
Update: I re-implemented this in a more robust way.